src/TreeDumpLibrary/VisualTreeDumper.cs

```diff
@@ -333,6 +335,32 @@ public string PropertyValueToString(string propertyName, object propertyObject)
                 Size size = (Size)propertyObject;
                 return $"[{(int)size.Width}, {(int)size.Height}]";
             }
+            else if(propertyObject is IList<TextHighlighter>)
+            {
+                var sb = new StringBuilder();
+                sb.Append("[");
+
+                foreach (var th in propertyObject as IList<TextHighlighter>)
+                {
+                    var background = (SolidColorBrush)th.Background;
+                    sb.Append("\"(" + background.Color.ToString() + ",");
+
+                    var ranges = th.Ranges;
+                    foreach (var r in ranges)
+                    {
+                        sb.Append("(" + r.StartIndex + "," + r.Length + ")");
+                    }
+                    sb.Append(")\", ");
+                }
```

Comment:
If I'm following this correctly, this would produce a JSON blob for an array of strings

```json
[
  "(red, (5, 10))",
...
]

Consider instead producing this:
[
  {
       Background: "red",
       Start: "5",
       Length: "10"
  },
  {
    .... another text highlighter object ...
  }
]
```

Reply:
Yes, it currently produces a JSON array of strings:
"TextHighlighters": ["(#FF008000,(15,18))", "(#FF008000,(33,1))", "(#FF0000FF,(34,18))"],

The current implementation in RNW will result in 1 range per highlighter, but I might improve it later and they can come in as an array in the highlighter object, which would make it more complicated.
I did it this way, as it seemed that I'd basically need to replicate the parsing logic for "children" from a level above and apply it to PropertyValueToString(). I wasn't sure if I'd get the indentation right, being past the call to BeginNode().

Should I create the suggested output manually within this function?